great_expectations_cloud/agent/agent.py

```diff
@@ -282,8 +285,11 @@ def _get_config(cls) -> GXAgentConfig:
                 gx_cloud_access_token=env_vars.gx_cloud_access_token,
             )
         except pydantic.ValidationError as validation_err:
-            raise GXAgentError(
-                f"Missing or badly formed environment variable\n{validation_err.errors()}"
+            missing_variables = ", ".join(
+                [f"{validation_error['loc'][0]}" for validation_error in validation_err.errors()]
+            )
+            raise GXAgentConfigError(
+                f"Missing or badly formed environment variable(s). Make sure to set the following environment variable(s): {missing_variables}"
             ) from validation_err
```

Comment:
good call, thanks - I created a logging_util.py file that handles this now. And also unit tests for that file.

Reply:
Thank you! I was originally thinking of a private method in `GXAgent` but this works too! What do you think about moving the function into `great_expectations_cloud/agent/agent.py` as a module level function instead of a util file? I tend to shy away from util files since they can become cluttered and disconnected from the context where they are used. 
Here is a pretty good [writeup on util files](https://softwareengineering.stackexchange.com/a/422932) with some add'l rationale.

Reply:
Thanks; saving that stackoverflow link :) 

As discussed, I moved this code to the config.py file since it's a pretty print method that's specifically used for printing config validation errors. (Also renamed it.)

There are a couple of other places we can use this method too where we are checking config; I'll add those in another PR.